### PR TITLE
Accept stop signal

### PR DESCRIPTION
**Pull request by jluhrsen**
One problem this fixes is during a node reboot when a SIGTERM
is given to the container it will stop in a timely fashion.
without this, the node has to wait for it's timeout (e.g., 30s)
before giving up and SIGKILL'ing it.

Signed-off-by: Jamo Luhrsen <jluhrsen@gmail.com>

### CODE REVIEW

**Comment by jluhrsen** [comment]
/hold
for testing

**Review by adambkaplan — APPROVED** [review]
/approve

Looks good to me, I would like a second set of eyes on this for lgtm.

**Comment by jluhrsen** [comment]
@adambkaplan, looking through my open items and saw this. anyone specific we can ping to get a double check?

**Comment by adambkaplan** [comment]
/retest

I think this is okay, I want to ensure that the upgrade test passes before merging.

**Comment by jluhrsen** [comment]
> /retest
> 
> I think this is okay, I want to ensure that the upgrade test passes before merging.

this upgrade job is being difficult. most of the retests seem to fail on infra related issues and not even getting to the installed cluster and running tests/upgrades. looking at [testgrid](https://testgrid.k8s.io/redhat-openshift-ocp-release-4.11-informing#periodic-ci-openshift-release-master-ci-4.11-e2e-aws-upgrade) and [sippy](https://sippy.ci.openshift.org/sippy-ng/jobs/4.11/analysis?filters=%7B%22items%22%3A%5B%7B%22columnField%22%3A%22name%22%2C%22operatorValue%22%3A%22equals%22%2C%22value%22%3A%22periodic-ci-openshift-release-master-ci-4.11-e2e-aws-upgrade%22%7D%5D%2C%22linkOperator%22%3A%22and%22%7D) I can see that it's not the most healthy job anyway. will keep
trying a retest a few more times to see if it will pass eventually.

**Comment by jluhrsen** [comment]
> > /retest
> > I think this is okay, I want to ensure that the upgrade test passes before merging.
> 
> this upgrade job is being difficult. most of the retests seem to fail on infra related issues and not even getting to the installed cluster and running tests/upgrades. looking at [testgrid](https://testgrid.k8s.io/redhat-openshift-ocp-release-4.11-informing#periodic-ci-openshift-release-master-ci-4.11-e2e-aws-upgrade) and [sippy](https://sippy.ci.openshift.org/sippy-ng/jobs/4.11/analysis?filters=%7B%22items%22%3A%5B%7B%22columnField%22%3A%22name%22%2C%22operatorValue%22%3A%22equals%22%2C%22value%22%3A%22periodic-ci-openshift-release-master-ci-4.11-e2e-aws-upgrade%22%7D%5D%2C%22linkOperator%22%3A%22and%22%7D) I can see that it's not the most healthy job anyway. will keep trying a retest a few more times to see if it will pass eventually.

@adambkaplan , finally got upgrade job to pass.

**Review by adambkaplan — APPROVED** [review]
/lgtm

**Pull request merged**
